Sources/AppStoreServerLibrary/SignedDataVerifier.swift

```diff
@@ -3,9 +3,9 @@
 import Foundation
 
 ///A verifier and decoder class designed to decode signed data from the App Store.
-public struct SignedDataVerifier {
+public struct SignedDataVerifier: Sendable {
```

Comment:
`SignedDataVerifier` must be sendable in order for it to be used by multiple concurrent tasks at the same time. This means that `ChainVerifier` must be updated to an actor to properly prevent data race issues.